pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -1026,35 +1298,82 @@ def export_mesh_obj(self, setup_name=None, intrinsic_dict={}):
         return None
 
     @aedt_exception_handler
-    def plot_model_obj(self, export_afterplot=True, jupyter=False):
-        """Plot the model.
+    def plot_model_obj(
+        self,
+        objects=None,
+        export_afterplot=True,
+        export_path=None,
+        plot_separate_objects=True,
+        air_objects=False,
+        show_axes=True,
+        show_grid=True,
+        show_legend=True,
+        background_color="white",
+        object_selector=True,
+        windows_size=None,
+        off_screen=False,
+        color=None,
+    ):
+        """Plot the model or a substet of objects.
 
         Parameters
         ----------
-        export_afterplot : bool, optional
-             Whether to export the plot after it is generated. The
-             default is ``True``.
-        jupyter : bool, optional
-             Whether to generate the plot using Jupyter Notebook.  The default is ``False``.
+        objects : list
+            Optional list of objects to plot. If `None` all objects will be exported.
+        export_afterplot : bool
+            Set to True if the image has to be exported after the plot is completed.
+        export_path : str
+            File name with full path. If `None` Project directory will be used.
+        plot_separate_objects : bool
+            Plot each object separately. It may require more time to export from AEDT.
+        air_objects : bool
+            Plot also air and vacuum objects.
+        show_axes : bool
+            Define if axis will be visible or not.
+        show_grid : bool
+            Define if grid will be visible or not.
+        show_legend : bool
+            Define if legend is visible or not.
+        background_color : str, list
+            Define the plot background color. Default is `"white"`.
+            One of the keys of `pyaedt.generic.constants.CSS4_COLORS` can be used.
+        object_selector : bool
+            Enable the list of object to hide show objects.
+        windows_size : list
+            Windows Plot size.
+        off_screen : bool
+            Off Screen plot
```

Comment:
```suggestion
        objects : list, optional
            Optional list of objects to plot. If `None` all objects will be exported.
        export_afterplot : bool, optional
            Set to True if the image has to be exported after the plot is completed.
        export_path : str, optional
            File name with full path. If `None` Project directory will be used.
        plot_separate_objects : bool, optional
            Plot each object separately. It may require more time to export from AEDT.
        air_objects : bool, optional
            Plot also air and vacuum objects.
        show_axes : bool, optional
            Define if axis will be visible or not.
        show_grid : bool, optional
            Define if grid will be visible or not.
        show_legend : bool, optional
            Define if legend is visible or not.
        background_color : str, optional
            Define the plot background color. Default is `"white"`.
            One of the keys of `pyaedt.generic.constants.CSS4_COLORS` can be used.
        object_selector : bool, optional
            Enable the list of object to hide show objects.
        windows_size : list, optional
            Windows Plot size.
        off_screen : bool, optional
            Off Screen plot
```